format/midi/testdata/format-0.fqtest

```diff
@@ -0,0 +1,269 @@
+0x090|                                             ff|               .|          event: "tempo" (65361) 0x9f-0xa1 (2)
+0x0a0|51                                             |Q               |
+0x0a0|   03 07 a1 20                                 | ...            |          tempo: 500000 0xa1-0xa5 (4)
+     |                                               |                |        [13]{}: smpte_offset 0xa5-0xae (9)
+     |                                               |                |          time{}: 0xa5-0xa6 (1)
+0x0a0|               00                              |     .          |            delta: 0 0xa5-0xa6 (1)
+     |                                               |                |            tick: 0
+0x0a0|                  ff 54                        |      .T        |          event: "smpte_offset" (65364) 0xa6-0xa8 (2)
+     |                                               |                |          offset{}: 0xa8-0xae (6)
+0x0a0|                        05 4d 2d 3b 07 27      |        .M-;.'  |            bytes: "[77 45 59 7 39]" 0xa8-0xae (6)
+     |                                               |                |            framerate: 25 (1) 0xae-0xae (0)
+     |                                               |                |            hour: 13
+     |                                               |                |            minute: 45
+     |                                               |                |            second: 59
+     |                                               |                |            frames: 7
+     |                                               |                |            fractions: 39
+     |                                               |                |        [14]{}: time_signature 0xae-0xb6 (8)
+     |                                               |                |          time{}: 0xae-0xaf (1)
+0x0a0|                                          00   |              . |            delta: 0 0xae-0xaf (1)
+     |                                               |                |            tick: 0
+0x0a0|                                             ff|               .|          event: "time_signature" (65368) 0xaf-0xb1 (2)
+0x0b0|58                                             |X               |
+     |                                               |                |          signature{}: 0xb1-0xb6 (5)
+0x0b0|   04 04 02 18 08                              | .....          |            bytes: "[4 2 24 8]" 0xb1-0xb6 (5)
+     |                                               |                |            numerator: 4
+     |                                               |                |            denominator: 4
+     |                                               |                |            ticks_per_click: 24
+     |                                               |                |            thirty_seconds_per_quarter: 8
```

Comment:
Would be nice in cases like this that the fields could be decode to get proper bit ranges, that is usually how i want fq to work. That would make the "bytes" fields not needed and the raw bytes would still be accessable using `... | .signature | tobytes` and you can use `explode` to get a array of the bytes. Is there particular reason you want to show `[decimal decimal ...]`?

Reply:
Fixed in [578b7e78e395f4c1baefd8adb2cd6794176f2d2b](https://github.com/wader/fq/pull/1004/commits/578b7e78e395f4c1baefd8adb2cd6794176f2d2b)